2023-risks/m2-insecure-communication.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: col-sidebar
+title: "M2: Insecure Communication"
+---
+
+# Threat Agents
+
+**Application Specific**
+
+Most modern mobile applications exchange data with one or more remote servers. When the data transmission takes place, it typically goes through the mobile device's carrier network and the internet, a threat agent listening on the wire can intercept and modify the data if it transmitted in plaintext or using a deprecated encryption protocol. The following threat agents exist:
+
+- An adversary that shares your local network (compromised or monitored Wi-Fi);
+- Rogue carrier or network devices (routers, cell towers, proxy's, etc); or
+- Malware on your mobile device.
```

Comment:
I would add examples for the agents' motivation, as without motivation they would not be a threat.

Reply:
That would be the case in a threat intelligence context, here we're mostly interested in adversary capabilities to compromise the secure communication regardless of their motivation.

Reply:
Nevertheless, I think it would be helpful to have such examples to make clear that motivation is also a prerequisite for an attack to happen, because I would assume that most readers are not aware of this distinction. Otherwise, one might naively think that everyone with a capability is also a threat